roles/confluent.variables/vars/main.yml

```diff
@@ -580,7 +584,7 @@ kafka_connect_properties:
   rbac:
     enabled: "{{rbac_enabled}}"
     properties:
-      rest.extension.classes: "io.confluent.connect.security.ConnectSecurityExtension{% if kafka_connect_secret_registry_enabled|bool %},io.confluent.connect.secretregistry.ConnectSecretRegistryExtension{% endif %}"
+      rest.extension.classes: "{% if kafka_connect_extension_classes_custom | length >0 %}{{ kafka_connect_extension_classes_custom }},{% endif %}io.confluent.connect.security.ConnectSecurityExtension{% if kafka_connect_secret_registry_enabled|bool %},io.confluent.connect.secretregistry.ConnectSecretRegistryExtension{% endif %}"
```

Comment:
this if statement isnt necessary right? if the string has length 0 then just concatenate it

Reply:
its only purpose is to keep the comma in place after the custom classes.

Reply:
Oh you're so right!

Reply:
I'm not really enjoying all the if statements, this is a solution i came up with on a different PR that uses list objects:

```
# in roles/confluent.variables/defaults/main.yml
### Additional set of comma-separated extension classes
kafka_connect_extension_classes_custom: ""


# in roles/confluent.variables/vars/main.yml
kafka_connect_rest_extension_classes:
  - "{% if rbac_enabled|bool %}io.confluent.connect.security.ConnectSecurityExtension{% endif %}"
  - "{% if kafka_connect_secret_registry_enabled|bool %},io.confluent.connect.secretregistry.ConnectSecretRegistryExtension{% endif %}"
 
kafka_connect_final_rest_extension_classes: "{{kafka_connect_rest_extension_classes| difference(['']) + kafka_connect_custom_rest_extension_classes.split(',')}}"

  rest_classes:
    enabled: "{{ kafka_connect_final_rest_extension_classes|length > 0 }}"
    properties:
      rest.extension.classes: "{{ kafka_connect_final_rest_extension_classes.join(',') }}"
```

WDYT?